conductr_cli/validation.py

```diff
@@ -219,11 +219,7 @@ def handler(*args, **kwargs):
             log = get_logger_for_func(func)
             log.error('Docker native is installed and Docker machine environment variables are set.')
             log.error('It is uncertain which Docker VM should be used.')
-            log.error('If Docker native should be used please unset the Docker machine environment variables:')
-            log.error('  DOCKER_CERT_PATH')
-            log.error('  DOCKER_HOST')
-            log.error('  DOCKER_MACHINE_NAME')
-            log.error('  DOCKER_TLS_VERIFY')
+            log.error('If Docker native should be used please unset the DOCKER_MACHINE_NAME environment variable.')
```

Comment:
If `DOCKER_MACHINE_NAME` is set then we know that Docker machine is enabled and that the user needs to unset all 4 envs. So the solution here is to keep the old error message, but still only do the check on `DOCKER_MACHINE_NAME`.